src/tests/Login.test.jsx

```diff
@@ -107,3 +107,25 @@ it('handles wrong credentials', async () => {
     
     expect(screen.getByLabelText('errorMessage')).toHaveTextContent("The server is currently unavailable. Try again later")
   })
+
+it('Checking if the fields are empty', async () => {
+
+    const{getByLabelText} = render(<Login />)
+
+    fireEvent.change(screen.getByLabelText("Username or Email:", {selector: "input"}), {target: { value: "" }})
+    
+    fireEvent.change(screen.getByLabelText("Password :", {selector: "input"}), {target: { value: "" }})
+    
+    act(() => {
+
+        expect(getByLabelText("Username or Email:", {selector: "input"})).toHaveValue("");
```

Comment:
expect is an assert, so not part of `act` so put it below line 125 `act` block (not inside the block itself). You can read more about 3As (Arrange, Act, and Assert) [here](https://testing-library.com/docs/react-testing-library/example-intro).
Also, the test still failed on Password field. Can you fix this?
<img width="1680" alt="Screen Shot 2020-09-02 at 6 20 42 pm" src="https://user-images.githubusercontent.com/29667122/91958723-10e51480-ed4b-11ea-9cfb-3e02034bdf63.png">

Reply:
Don't forget we want to check the `required` attribute, so please use .toBeRequired() instead of .toHaveValue("")

Reply:
yeah sure....i have done that now please check. thank you

Reply:
i got the problem why that password field was failing